templates/about/release-cycle.html

```diff
@@ -10,380 +10,541 @@
-        </p>
-
-        <p>
-          For <a href="/support">Ubuntu Pro</a> support, users must upgrade to remain in the N-2 support window.
-        </p>
-
-        <p>
-          ESM support includes fixes to high and critical CVEs related to Kubernetes charms and MicroK8s (snap), in the current N-4 support window. Upstream updates and fixes to Kubernetes binaries or container images are absorbed by Charmed Kubernetes and MicroK8s.
-        </p>
-
-        <p>The Charmed Kubernetes and MicroK8s support lifecycle can be represented this way:</p>
-      </div>
-
-      {% include "/about/release_cycles/k8s-eol.html" %}
-
-      <div class="u-fixed-width">
-        <p>
-          For more information on previous and current releases, please see the <a href="/kubernetes/charmed-k8s/docs/release-notes">Charmed Kubernetes release notes</a> or the <a href="https://microk8s.io/docs/release-notes">MicroK8s release notes.</a>
-        </p>
       </div>
     </div>
   </section>
 
-  <section class="p-tabs__content"
-           id="canonical-ceph-release-cycle"
-           role="tabpanel"
-           aria-labelledby="canonical-ceph-release-cycle-tab">
-    <div class="p-strip is-shallow">
-      <div class="u-fixed-width">
-        <h2>Canonical Ceph release cycle</h2>
-        <p>
-          For the Ceph release cycle and supported versions, refer to the <a href="/ceph/docs/supported-ceph-versions">product documentation</a>.
-        </p>
+  <section class="p-section--deep">
+    {{ vf_basic_section(
+      title={"text": "Canonical Ubuntu lifecycle
+            <br />
+            and release cadence key concepts"},
+      items=[
+        {
+          "type": "description",
+          "item": {
+            "type": "text",
+            "content": "Canonical publishes new releases of Ubuntu and other products in our portfolio on a regular cadence, enabling the community, businesses and developers to plan their roadmaps with the certainty of access to newer open source upstream capabilities."
+          }
+        }
+      ]
+    ) }}
+    <div class="grid-row">
+      <div class="grid-col-start-large-3 grid-col-6">
+        <div class="p-section--shallow">
+          <nav class="p-tabs js-tabbed-content">
+            <ul class="p-tabs__list u-no-margin--bottom"
+                data-maintain-hash="true"
+                role="tablist">
+              <li class="p-tabs__item" role="presentation">
+                <a class="p-tabs__link"
+                   id="ubuntu-releases-tab"
+                   role="tab"
+                   aria-controls="ubuntu-releases"
+                   aria-selected="true">Ubuntu releases</a>
```

Comment:
This is missing an href.
This makes the link not tabbable.

https://github.com/user-attachments/assets/049f181b-9849-4bd0-9e5d-a90b5434d6fb